wgsl/index.bs

```diff
@@ -4886,9 +4886,34 @@ assignment_statement
       which maps to OpAccessChain followed by OpStore
 </pre>
 
-## Control flow TODO ## {#control-flow}
+## Control Flow ## {#control-flow-section}
 
-### Sequence TODO ### {#sequence-statement}
+<dfn>Control flow</dfn> is the ordering of statement execution.
+
+### Sequenced Execution ### {#sequence-statement}
+
+Statements within a compound statement execute in textual order, unless interrupted
+by a [=control flow interruption=].
+More precisely:
+
+<div algorithm="executing a compound statement">
+    A [=compound statement=] |C| executes as follows:
+
+    * If |C| is empty, then it immediately finishes execution.
+    * Otherwise:
+        * The first statement in the compound statement becomes the |Current| statement.
+        * Repeatedly execute a |Current| statement:
+            * Execute the |Current| statement.
+            * If |Current| is a [=control flow interruption=], then control has been transferred as prescribed
```

Comment:
Thinking about this more, I realized this doesn't capture the case where a discard statement is executed somewhere inside a called function.   I'll have to reword this.

Reply:
Fixed in the amended commit.